consensus/safety-rules/src/safety_rules.rs

```diff
@@ -303,9 +266,42 @@ impl SafetyRules {
 
             info!(SafetyLogSchema::new(LogEntry::Epoch, LogEvent::Update).epoch(epoch_state.epoch));
         }
-        self.epoch_state = Some(epoch_state);
+        self.epoch_state = Some(epoch_state.clone());
 
-        Ok(())
+        let author = self.persistent_storage.author()?;
```

Comment:
This would flag that secure storage is unavailable, we don't mask the error in safety rules. @JoshLind / @aching